src/app/pages/ArticlePage/ElectionBanner/config.ts

```diff
@@ -0,0 +1,13 @@
+import { getEnvConfig } from '#app/lib/utilities/getEnvConfig';
+
+const IFRAME_BASE_URL = getEnvConfig().SIMORGH_INCLUDES_BASE_URL;
+
+export default {
+  iframeHeight: 324,
+  iframeSrc: `${IFRAME_BASE_URL}/include/vjafwest/1365-2024-us-presidential-election-banner/develop/{service}/app`,
+  iframeSrcAmp: `${IFRAME_BASE_URL}/include/vjafwest/1365-2024-us-presidential-election-banner/develop/{service}/app/amp`,
+  iframeTitle: 'US Election',
```

Comment:
Do we care about translations? If so, we could get it from the BBC Things website if needed

Reply:
We are aiming to translate the title from editorial, but its a good point as the thingLabel gives a translate label for us. Let me check with editorial.

Reply:
Messaged editorial, but I've changed it to using the `thingLabel` as it leaves us in a better position anyway.

Reply:
All good from editorial. Thanks for the tip!